tox.ini

```diff
@@ -12,7 +12,7 @@ deps =
 [flake8]
 ignore = W504, E501
 
-[testenv:mypy]
+[mypy]
```

Comment:
```suggestion
[tentenv:mypy]
```

I think it does in fact need the `testenv`

Reply:
Yes. I'm fumbling a bit to figure out why `tox -v` runs mypy locally, but not here in Travis. I the change.